src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -32,6 +60,209 @@ class RoutePathView extends React.Component{
+                        disabled={this.state.isEditingDisabled}
+                    />
+                    <InputContainer
+                        label='PÄÄTEPAIKKA SUOMEKSI'
+                        disabled={this.state.isEditingDisabled}
+                    />
+                    <InputContainer
+                        label='PÄÄTEPAIKKA RUOTSIKSI'
+                        disabled={this.state.isEditingDisabled}
+                    />
+                    <InputContainer
+                        label='LYHENNE RUOTSIKSI'
+                        disabled={this.state.isEditingDisabled}
+                    />
+                    <div className={s.flexInnerRowAligned}>
+                        <InputContainer
+                            label='PITUUS'
+                            disabled={this.state.isEditingDisabled}
+                        />
+                        <div className={s.calculateButtonContainer}>
+                            <Button
+                                onClick={this.onChange}
+                                type={ButtonType.ROUND}
+                                text={'Laske'}
+                            />
+                        </div>
+                    </div>
+                    <div className={s.inputContainer}>
+                        <div className={s.subTopic}>
+                            POIKKEUSREITTI
+                        </div>
+                        <div className={s.checkBoxContainer}>
+                            <div className={s.flexInnerRow}>
+                                <div className={s.flexInnerRow}>
+                                    <Checkbox
+                                        checked={false}
+                                        text={'Ei'}
+                                        onClick={this.onChange}
+                                    />
+                                </div>
+                                <div className={s.flexInnerRow}>
+                                    <Checkbox
+                                        checked={false}
+                                        text={'Kyllä'}
+                                        onClick={this.onChange}
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className={s.flexRow}>
+                <InputContainer
+                    label='PÄIVITYSPVM'
+                    disabled={this.state.isEditingDisabled}
+                />
+                <InputContainer
+                    label='PÄIVITTÄJÄ'
+                    disabled={this.state.isEditingDisabled}
+                />
+            </div>
+            <div className={s.sectionDivider}/>
+            <div className={s.flexRow}>
+                <div className={s.flexInnerColumn}>
+                    <div className={s.inputContainer}>
```

Comment:
Why having input container wrapper around each button? Just for padding? Add padding to .button by default? Or to .routePathView .button?